src/opnsense/mvc/app/library/OPNsense/Base/UIModelGrid.php

```diff
@@ -125,9 +125,24 @@ public function fetch(
                 // before searching.
                 $row = [];
                 $row['uuid'] = $record->getAttributes()['uuid'];
-                foreach ($fields as $fieldname) {
-                    if ($record->$fieldname != null) {
-                        $row[$fieldname] = $record->$fieldname->getDescription();
+                $content = $record->getNodeContent();
+                foreach ($content as $field => $val) {
+                    if (str_starts_with($field, '$') && in_array($key = substr($field, 1), $fields)) {
+                        $row[$key] = $val;
+                        if ($content[$key] !== $val) {
+                            /**
+                             * getNodeContent() returns "$<key>" for description values
+                             * this would break API compat if we're returning this directly,
+                             * so swap the two around:
+                             *
+                             * [key] => "Descriptive (translated) value"
+                             * [$key] => "value used in config"
```

Comment:
I'm not opposed to the idea as already discussed. The question is why we cannot / should not switch those values while implementing this. The previous value (descriptive) doesn't have immediate operational value? A breaking change in e.g. 26.1 would seem reasonable?